### PR TITLE
Add a list of allowed_approvers to the config file

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

Adds a list of allowed_approvers to the config file. Also vastly improves the tests  
Note: This is based on the approval dates PR, so that should merge first

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver
- [ ] Add release notes to [unreleased changelog](https://github.com/teamsempo/SempoBlockchain/blob/master/CHANGELOG.md#unreleased)

### CODE REVIEW

**Comment by michielderoos** [comment]
[[Feedback] Approval process (6 hours)](https://trello.com/c/OVX2Z1Wl/826-feedback-approval-process-6-hours)

File: config.py [hunk 1]
```diff
@@ -120,6 +120,9 @@
 
 APP_HOST            = config_parser['APP']['APP_HOST']
 REQUIRE_MULTIPLE_APPROVALS = config_parser['APP'].getboolean('REQUIRE_MULTIPLE_APPROVALS', False)
+ALLOWED_APPROVERS = config_parser['APP'].get('allowed_approvers', None)
+if ALLOWED_APPROVERS:
+    ALLOWED_APPROVERS = ALLOWED_APPROVERS=ALLOWED_APPROVERS.split(',')
```
Comment (tristanhcole) [inline config.py:125]: Can we also add the config to the template https://github.com/teamsempo/SempoBlockchain/blob/master/config_files/public/local_config.ini

File: config.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 logging.basicConfig(level=env_loglevel)
 logg = logging.getLogger(__name__)
 
-VERSION = '1.9.11'  # Remember to bump this in every PR
+VERSION = '1.9.12'  # Remember to bump this in every PR
```
Comment (tristanhcole) [inline config.py:8]: I think this would be a feature bump?

File: app/test_app/functional/api/test_bulk_disbursement_api.py [hunk 3]
```diff
@@ -81,50 +82,243 @@ def test_disbursement(search_string, params, include_list, exclude_list, disburs
...
+def test_disbursement_approval_flow(include_list, disbursement_amount, expected_recipient_count, expected_total_disbursement_amount,\
+    response_status, recipient_firstnames, multi_approval_setting, allowed_approvers_setting, statuses, test_client, complete_admin_auth_token\
+    , create_organisation):
+    """
+    Tests the approval flow
+    If there are no allowed_approvers in the config, but multi_approval_setting is True, it takes two users to approve
+    If there are allowed_approvers in the config, and multi_approval_setting is True, it takes two users to approve and one must be in allowed_approvers
+    If multi_approval_setting is False, it just takes one to approve
+    """
+    # Utility function to create admins to approve
```
Comment (tristanhcole) [inline app/test_app/functional/api/test_bulk_disbursement_api.py:148]: Great tests!

**Review by tristanhcole — APPROVED** [review]
Tested & LGTM!

**Pull request merged**